### DISCUSSION TITLE
Welcome to apple-clone Discussions!

### DISCUSSION THREAD

**Discussion by sanidhyy** [sanidhyy/apple-clone#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ozzie-code-alt** [comment]
Did you follow a  tutorial for this one ?

**Reply by sanidhyy** [reply]
Yes, I followed a tutorial for this project. Here's the YouTube video by JS Mastery that I used: https://www.youtube.com/watch?v=kRQbRAJ4-Fs

**Reply by Ozzie-code-alt** [reply]
thank

> Yes, I followed a tutorial for this project. Here's the YouTube video by JS Mastery that I used: https://www.youtube.com/watch?v=kRQbRAJ4-Fs
thanks man
